src/App.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components/macro'
+import useFormData from './hooks/useFormData'
+import SaveGameForm from './components/SaveGameForm'
+import GameCard from './components/GameCard'
+
+
+function App() {
+
+  const { gameProfile, setGameProfile, savedGameProfiles, setSavedGameProfiles } = useFormData()
+ 
+  return (
+    <AppWrapper>
+      <SaveGameForm gameProfile={gameProfile} setGameProfile={setGameProfile} savedGameProfiles={savedGameProfiles} setSavedGameProfiles={setSavedGameProfiles} />
+      <GameCardListWrapper>
+        {savedGameProfiles.map(({location, date, players, winner, shots, _id}) => 
+          <GameCard key={_id} location={location} date={date} players={players} winner={winner} shots={shots}/>
```

Comment:
Why not just use a `savedGameProfile` object and pass it to `GameCard` instead of destructuring all properties and passing it individually?

Reply:
I was thinking of this, but decided to go with destructuring, in order to shorten line 16 with its attributes. So that I do not have to write ``{saveGameProfile.location}``and so on.